testbook/exceptions.py

```diff
@@ -1,7 +1,6 @@
 class TestbookError(Exception):
     """Generic Testbook exception class"""
-
-    pass
+    __test__ = False
```

Comment:
These keep pytest from complaining about there being no tests found in this class